contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -841,154 +841,6 @@ contract LegacyPooledStaking is IPooledStaking, MasterAwareV2, PricesV1 {
     return products;
   }
 
-  function createPoolAndDeposit(
```

Comment:
this is no longer used - it was for migration only.

As opposed to updating the code for the new structure to compile - i'd rather delete it.

Reply:
agreed